qemu/tests/cdrom.py

```diff
@@ -600,8 +600,8 @@ def test(self):
             error.context("Detecting the existence of a cdrom (qemu side)",
                           logging.info)
             qemu_cdrom_device = get_device(vm, iso_image)
-
-            self.session.get_command_output("umount %s" % guest_cdrom_device)
```

Comment:
1. I think the reason for umount as the Linux Guest w/ GUI will mount the cdrom autotmately ,umount it first ,then it can make sure it will not be locked .
2. I don't think windows and text mode Linux need this step.

Mike